.github/workflows/frontend-e2e-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Frontend E2E Workflow
+
+on:
+  push:
+    branches:
+      - master
+      - stage
+      - chore/add-e2e-pipeline
+
+jobs:
+  fe-e2e-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
```

Comment:
What do you think if we add a cache action here?
Like this:
https://dev.to/mattpocockuk/how-to-cache-nodemodules-in-github-actions-with-yarn-24eh

so we cache the npm modules, it should speed up the workflow time a bit if there are no changes to the yarn.lock

Reply:
We actually do not need to speed up because it should wait for Vercel deployment anyway. Any it takes like 5 minutes.
Actually, I have no idea how to sync deployment and the workflow..

Reply:
Is this helpful?
https://vercel.com/guides/how-can-i-run-end-to-end-tests-after-my-vercel-preview-deployment

Reply:
yep, this is looking good.